### ISSUE TITLE
Obsoletion request: GO:0006378 mRNA polyadenylation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#26944]
Please provide as much information as you can:

* **GO term ID and Label**

GO:0006378 mRNA polyadenylation

* **Reason for deprecation** Put an x in the appropriate box: 

 - [x] The reason for obsoletion is that this term represents a molecular function.

* **"Consider" term(s) (ID and label)**

If the annotation is mRNA cleavage and polyadenylation specificity factor complex
use
GO:0180010 | co-transcriptional mRNA 3'-end processing, cleavage and polyadenylation pathway

for others you will need to check the context (i.e. stability, or if TRAMP related, degradation)

* **Are there annotations to this term?**
- How many EXP: 28
2 ComplexPortal
1 EcoliWiki
8 FlyBase
16 GeneDB
1 WB

* **Are there mappings and cross references to this term? (InterPro, Keywords; check QuickGO cross-references section)**
4 interpro
2 unirule

* **Is this term in a subset? (check the AmiGO page for that term)**

* **Any other information**

----
Checklist for ontology editor

***Check term usage and metadata in Protégé***
- [x] check term usage in the ontology 
- [x] check internal mappings: RHEA, EC, MetaCyc
- [x] check subset usage
- [x] check taxon constraints 

***Check annotations***

- [x] https://github.com/geneontology/go-annotation/issues/4983
- [x] find external mappings (via IEAS), include in obsoletion notice  
- [x] tag curators in Annotation Review ticket. Responsible annotator for each group: https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv

***Notification***
- [x] create [obsoletion announcement](https://github.com/geneontology/go-announcements/issues/new?assignees=&labels=obsoletion&template=obsoletion-notice.md&title=Obsoletion+notice%3A+%5BGO+ID%3A+term+label%5D)
- [x] announce to GO friends (go-friends@mailman.stanford.edu)

**Comment by pgaudet** [comment]
Is this really a single step ? 
https://en.wikipedia.org/wiki/Polyadenylation
> The [processive](https://en.wikipedia.org/wiki/Processivity) polyadenylation complex in the nucleus of eukaryotes works on products of [RNA polymerase II](https://en.wikipedia.org/wiki/RNA_polymerase_II), such as [precursor mRNA](https://en.wikipedia.org/wiki/Precursor_mRNA). Here, a multi-protein complex (see components on the right)[[18]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-bienroth-18) cleaves the 3′-most part of a newly produced RNA and polyadenylates the end produced by this cleavage. The cleavage is catalysed by the enzyme [CPSF](https://en.wikipedia.org/wiki/CPSF)[[13]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-histone-13)[[18]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-bienroth-18) and occurs 10–30 nucleotides downstream of its binding site.[[19]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-Liu-19)  (...) 
 **Two other proteins add specificity to the binding to an RNA: CstF and CFI.** CstF binds to a GU-rich region further downstream of CPSF's site.[[21]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-Beaudoing-21) CFI recognises a third site on the RNA (...) 
**[Polyadenylate polymerase**](https://en.wikipedia.org/wiki/Polyadenylate_polymerase) builds the poly(A) tail by adding [adenosine monophosphate](https://en.wikipedia.org/wiki/Adenosine_monophosphate) units from [adenosine triphosphate](https://en.wikipedia.org/wiki/Adenosine_triphosphate) to the RNA, cleaving off [pyrophosphate](https://en.wikipedia.org/wiki/Pyrophosphate).[[32]](https://en.wikipedia.org/wiki/Polyadenylation#cite_note-32) Another protein, PAB2, binds to the new, short poly(A) tail and increases the affinity of polyadenylate polymerase for the RNA.

**Comment by ValWood** [comment]
This  is a process, it is represented by
"co-transcriptional RNA 3'-end processing, cleavage and polyadenylation pathway"

See
https://www.pombase.org/term/GO:0180012
covers all of the pathway steps, cleavage, polyadenylation and other activities.
(I checked this label with experts when it was added back in  Feb '23). It's the last part of co-transcriptional processing,  pathway we were looking at yesterday. 

But there are lots of other polyadenylation activities (currently mostly represented by the pathways   "polyadenylation dependent blah". These will need reviewing and better unambiguous term labels (i.e TRAMP-dependent catabolic process). I will do this once co-transcriptional part is done.

I kept the "cytoplasmic mRNA polyadenylation" term for now because there is no obvious pathway to represent that, but the grouping term "mRNA polyadenylation" is no longer required.

Essentially we want the pathways, and no pathway is just "polyadenylation"
